### PR TITLE
Updating deployer-brooklyn-tosca artifact.

### PR DESCRIPTION
**Pull request by kiuby88**
Updating deployer-brooklyn-tosca artifact

### CODE REVIEW

File: byon/files/seaclouds-catalog.bom [hunk 1]
```diff
@@ -185,7 +185,7 @@ brooklyn.catalog:
         brooklynnode.webconsole.nosecurity: true
         brooklynnode.classpath:
           - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.9.0-20162302.17.14.52.jar
-          - https://s3-eu-west-1.amazonaws.com/tosca-temporal-integration/brooklyn-tosca-transformer-0.9.0-SNAPSHOT-20162401.225903.jar
+          - https://s3-eu-west-1.amazonaws.com/tosca-temporal-integration/deployer-brooklyn-tosca-transformer-0.9.0-20162602.100347.jar
```
Comment (andreaturli) [inline byon/files/seaclouds-catalog.bom]: Pleas add somewhere the doc to generate this jar

Comment (kiuby88) [inline byon/files/seaclouds-catalog.bom]: @andreaturli, sorry for my late response
I have written a first README version

**Comment by andreaturli** [comment]
thanks @kiuby88

**Comment by kiuby88** [comment]
Thanks @andreaturli

**Pull request merged**